runtime/lib/cloudapi/mqtt/client.js

```diff
@@ -83,6 +84,9 @@ MqttClient.prototype.onmessage = function onmessage (channel, message) {
     logger.error(err && err.stack)
     return
   }
+  if (!msg.topic && config.prefixForUnknown) {
```

Comment:
Suggest this as a private member as `_prefixForUnknown`?

Reply:
The object `config` is from the json file `wormhole.json`, so the `prefixForUnknown` is a json object field actually.

Reply:
Yea, what I meant is use a underscore prefix to represent as it's private?

Reply:
Updated.